src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/styled.ts

```diff
@@ -14,6 +14,7 @@ export const StyledFormFooter = styled(FormFooter)`
 
 export const StyledLabel = styled(Label)`
   font-weight: 400;
+  margin-top: -${themeSpacing(2.25)}; //first child has margin-top of 9px but distance to above element needs to be 24px
```

Comment:
Three orange flags I see here: I would avoid min margins, 2.25 themespacing and comments for styling. 
Tackle it at the root. So in this case the `margin-bottom: ${themeSpacing(6)};` on the RightSection doesn't doe the job. Instead, give every element it's own margin that suits the job, if Linda insists. Because the same behaviour already exists on the Standard Texts.

Reply:
yeah agree, maybe change the border of the component above

Reply:
Hiya,
talked to Linda, she didn't like the idea of giving every element its own margin, so we agreed I would remove the margin correction on this specific element. I see the flags, but if you need to correct 9 px I don't see how you can avoid 2.25 themespacing (except for using px but in my mind that's even less desirable)